processor/tailsamplingprocessor/testdata/tail_sampling_config.yaml

```diff
@@ -2,6 +2,8 @@ tail_sampling:
   decision_wait: 10s
   num_traces: 100
   expected_new_traces_per_sec: 10
+  decision_cache:
+    sampled_cache_size: 500
```

Comment:
If our recommendation is to keep the number of items an order of magnitude higher, we should make this way bigger than the num traces in this example.